pkg/user/tx_client.go

```diff
@@ -39,6 +40,14 @@ const (
 
 type Option func(client *TxClient)
 
+// TxResponse is a response from the chain after
+// a transaction has been submitted.
+type TxResponse struct {
+	Height int64
+	TxHash string
+	Code   uint32
+}
```

Comment:
what's the benefit of having our own type here? if this is different from the sdk TxResponse, is there a way to indicate it in the name?